src/js/image-select.js

```diff
@@ -0,0 +1,115 @@
+import classNames from "classnames";
+
+import ImageContent from "./image-content";
+import buzz from "../sounds/buzz.mp3";
+import chime from "../sounds/chime.mp3";
+
+const ANIMATION_DURATION = 500;
+
+const ImageSelect = React.createClass({
+  getInitialState: () => ({
+    currentIndex: 0
+  }),
+
+  handleSelect(id, answer) {
+    const { selected } = this.state;
+
+    this.setState({ selected: id, wordShow: false }, () => {
+      if (id === answer) {
+        this.playSound("chime");
+
+        window.setTimeout(this.proceedToNext, ANIMATION_DURATION);
+      } else {
+        this.playSound("buzz");
+        window.setTimeout(
+          () => this.setState({ selected: null }),
+          ANIMATION_DURATION
+        );
+      }
+    });
+  },
+
+  playSound(name) {
+    this[name].currentTime = 0;
+    this[name].volume = 0.6;
+    this[name].play();
+  },
```

Comment:
i'm not sure about browser compatibility here. are you confident this will work on older mobile browsers?

Reply:
Yeah, this should not be a problem https://caniuse.com/#search=audio

Reply:
interesting, so webpack is creating an audio tag and exposing its api for you to use? if so, sgtm (sounds good to me)

Reply:
Not webpack but React Refs: https://reactjs.org/docs/refs-and-the-dom
Refer: L107 & 108 in the same file

Reply:
ohhhhh I see what you did there. I missed where you actually just used an audio tag and kept the ref. seems alright to me.